docs/personalization/filters.md

```diff
@@ -1,136 +1,144 @@
+
+You can provide multiple reference categories (both in the request and by defining 
+a context item or items).
+In such a case, the superset of the recommendations is returned, and the results 
+are sorted based on a global weight of the recommendations. 
+Depending on the popularity of the categories, the more popular categories push 
+the less popular categories out of the results.
 
-|Filter|Requirements and restrictions|
-|---|---|
-|Editor blacklisted products.|Advanced edition gets several editor-based models. For more information, see [6. Recommendation Models](recommendation_models.md). One of them is an editor blacklist. The products from the blacklist will always be removed from every list of recommendations. You can use this feature to remove some standard products which users will buy in any case without recommendation (bread, butter, milk) or non-existing test products which are only used by a robot to check if the shop software is working properly.|
-|Customer blacklisted products|In addition to the standard set the website can send blacklist events for selected products and users. It can be implemented as e.g. a button "I do not like this product". If a user added a product to their blacklist, it will not be recommended to them anymore.|
-|Product validity interval|Additional product information [can be exported to the recommendation engine](content_import.md). If the product information is available and contains a validity time interval (for example the product is available only until 24 December 2014), it will be used to remove invalid products from the recommendation list.|
+If the recommended item is located in more than one category, it is recommended 
+that at least one category is requested in the recommendation call.
 
-## Boost-Filters
+### Multiple category path dimensions for popularity models
 
-In addition to general filters, which remove specified elements from recommendations, 
-the boost filter allows shifting (boosting) selected recommended items to the top positions of 
-the recommendation response. 
-This may happen if a selected attribute of the customer has the same value as the selected attribute 
-of the recommended item. 
-For example, we can show the news from the user's home country with higher priority than the rest.
+The category path parameter is a powerful tool. 
+A typical approach is to represent the default content in the navigation-based 
+structure of a website. 
+If you need to represent available items of a website in different dimensions (taxonomies), 
+you can do this out-of-the box, by enriching the `categorypath` information of an item.
 
-Assuming every item in the picture below has the attribute "country" and the user has the attribute 
-"country\_of\_origin", the boost filter can be configured in **Filter settings**, 
-which leads to boost recommendations for certain users:
+For example, the shop mentioned above sells both furniture and plants, and products 
```

Comment:
"shop mentioned above" - it either isn't mentioned, or too far away to refer to it